plugins/ipam/host-local/main.go

```diff
@@ -54,8 +54,15 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
-	r := &types.Result{
-		IP4: ipConf,
+	r := &types.Result{}
+	if ipConf.IP.IP.To4() != nil {
+		r = &types.Result{
+			IP4: ipConf,
+		}
+	} else {
```

Comment:
Please don't make these mutually exclusive, as this will not allow for both to be set at the same time, which the spec supports.